code/__DEFINES/~monkestation/span.dm

```diff
@@ -4,3 +4,6 @@
 #define span_clockgray(str) ("<span class='clockgray'>" + str + "</span>")
 #define span_clockred(str) ("<span class='clockred'>" + str + "</span>")
 #define span_ratvar(str) ("<span class='ratvar'>" + str + "</span>")
+#define span_italics(str) ("<span class='italics'>" + str + "</span>")
+
+#define REQUEST_MENTORHELP "request_mentorhelp"
```

Comment:
already defined earlier in code, should be fine to delete. Tis whats failing the linters
```suggestion
```